Makefile

```diff
@@ -15,13 +15,13 @@ build-bundle:
 build-swagger:
 	cd swagger && npm run build
 test:
-	JEKYLL_NO_GITHUB=true bundle exec htmlproofer ./_site --assume-extension --check-html --allow-hash-href --empty-alt-ignore --only-4xx --disable-external --url_ignore "/esQuery\.config\.js/"
+	bundle exec htmlproofer ./_site --assume-extension --check-html --allow-hash-href --empty-alt-ignore --only-4xx --disable-external --url_ignore "/esQuery\.config\.js/"
 local:
-	JEKYLL_NO_GITHUB=true bundle exec jekyll serve --incremental
+	bundle exec jekyll serve --incremental
 jekyll-build:
-	JEKYLL_ENV=production bundle exec jekyll build
-deploy:
-	rsync -avP --rsync-path="sudo -u www-data rsync" _site/ developers.italia.it:/data/www/developers.italia.it/web/
+	bundle exec jekyll build
+deploy-vm:
+	rsync --delete -avP --exclude "Makefile" --rsync-path="sudo -u www-data rsync" _site/ developers.italia.it:/data/www/developers.italia.it/web/
```

Comment:
do we really need `--delete` ? process will slower. But we have to consider that it will be removed soon.